### PR TITLE
PAYARA-3107 Cleanup nucleus admin util (Payara4)

### PR DESCRIPTION
**Pull request by Cousjava**
…min-util

PAYARA-3107 cleanup nucleus admin util

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 JDK7 please

File: nucleus/admin/util/src/main/java/com/sun/enterprise/admin/commands/CreateSsl.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
+// Portions Copyright [2018] Payara FOundatin and/or affiliates
```
Comment (MarkWareham) [inline nucleus/admin/util/src/main/java/com/sun/enterprise/admin/commands/CreateSsl.java]: typo

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 please

**Comment by Cousjava** [comment]
Jenkins, quick test Payara 4 JDK7 please

**Pull request merged**